1.1/terms_and_defns.ttl

```diff
@@ -0,0 +1,159 @@
+@prefix cs: <http://www.opengis.net/def/geosparql/td> .
+@prefix dcterms: <http://purl.org/dc/terms/> .
+@prefix owl: <http://www.w3.org/2002/07/owl#> .
+@prefix prof: <http://www.w3.org/ns/dx/prof/> .
+@prefix rdf: <http://www.w3.org/1999/02/22-rdf-syntax-ns#> .
+@prefix rdfs: <http://www.w3.org/2000/01/rdf-schema#> .
+@prefix sdo: <https://schema.org/> .
+@prefix skos: <http://www.w3.org/2004/02/skos/core#> .
+@prefix td: <http://www.opengis.net/def/geosparql/td/> .
+@prefix xsd: <http://www.w3.org/2001/XMLSchema#> .
+
+
+cs:
+    a owl:Ontology , skos:ConceptScheme ;
+    dcterms:created "2021-06-22"^^xsd:date ;
+	dcterms:creator "OGC GeoSPARQL Standards Working Group" ;
+    dcterms:modified "2021-06-22"^^xsd:date ;
+    dcterms:provenance "Each term (Concept) listed here indicates its own provenance since they are derived from other vocabularies and non-vocabulary sources."@en ;
+	dcterms:publisher [
+		a sdo:Organization ;
+		sdo:name "Open Geospatial Consortium" ;
+		sdo:url "https://www.ogc.org"^^xsd:anyURI ;
+	] ;
+    owl:versionInfo "OGC GeoSPARQL 1.1" ;
+    skos:definition "A vocabulary (taxonomy) of the Terms and Definitions listed by the GeoSPARQL 1.1 standard. These terms appear as a vocabulary resource (this file) and are also listed in Clause 4 of the GeoSPARQL 1.1 Specification"@en ;
+    skos:hasTopConcept
+        td:Subject ,
+        td:Predicate ,
+        td:Object ,
+        td:Resource ,
+        td:Class ,
+        td:Property ,
+        td:ObjectProperty ;
+        # td:DatatypeProperty ,
+        # td:AnnotationProperty 
+    skos:prefLabel "GeoSPARQL Terms & Definitions" ;
+.
+
+# Concept Template
+# td:XXX a skos:Concept ;
+#     dcterms:provenance ""@en ;
+#     rdfs:isDefinedBy <> ;
+#     skos:definition ""@en ;
+#     skos:inScheme cs: ;
+#     skos:prefLabel ""@en ;
+#     skos:topConceptOf cs: ;
```

Comment:
So all terms are to be top concepts? I suggest changing this to skos:broader, to link to one of the top concepts (semweb terms and GIS terms).

Reply:
No, see above. Can add in hierarchy where appropriate. Just haven't got too many obvious ones for now (other than the suggestion above)

Reply:
I've added hierarchy for at least the Resource & Property terms.

Reply:
See Commit https://github.com/opengeospatial/ogc-geosparql/pull/165/commits/09dbec599f1abc464554c98d7b2ddbc68054bef3